src/pages/components/DisabledByMdmView.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.0
```

Comment:
Move Camera's DisabledByMdmvView to the settings (same place as DisabledByMdmBanne) and use that here.

Just leave "text" property to the interface.

Reply:
Done. Used 'activity' as the text, as it's a parameter to the larger standard text.